ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/DynamicPivotBlockManager.java

```diff
@@ -68,8 +69,7 @@ public void check(final BiConsumer<BlockHeader, Boolean> onNewPivotBlock) {
                           .orElse(currentPivotBlockNumber);
               if (distanceNextPivotBlock > pivotBlockDistanceBeforeCaching
                   && isSearchingPivotBlock.compareAndSet(false, true)) {
-                syncActions
-                    .waitForSuitablePeers(FastSyncState.EMPTY_SYNC_STATE)
+                CompletableFuture.completedFuture(FastSyncState.EMPTY_SYNC_STATE)
                     .thenCompose(syncActions::selectPivotBlock)
```

Comment:
nit, but rather than composing a static completed future, couldn't this be reduced to 
`fastSyncActions.selectPivotBlock(fastSyncState)`